src/recommendations/RecommendationCard.jsx

```diff
@@ -20,9 +22,24 @@ const RecommendationCard = (props) => {
     return keys.slice(0, -2);
   };
 
+  const handleCardClick = () => {
+    trackRecommendationsClicked(
```

Comment:
Position of argus is not in correct order.

Reply:
good catch. thanks @mubbsharanwar